ts/DataGrid/Options.ts

```diff
@@ -198,6 +198,15 @@ export interface RowsSettings {
      * @default false
      */
     strictHeights?: boolean;
+
+    /**
+     * Rows virtualization option render rows that are visible in the viewport
+     * only. In case of large data set, the enabled option improve performance
+     * and saves memory.
+     *
+     * @default true
+     */
+    virtualization?: boolean;
```

Comment:
I think most cases should be covered with at standard version of the table.
IMO this should be disabled by default.

Reply:
On the one hand yes, on the other not. 
Let's consider the scenario: someone does not care how big data is, pushes data into our product for testing and... his first impression is that the DataGrid is slow or not smooth. 

Maybe I'm analyzing it too deep :D 

@stitot